eZ/Publish/Core/MVC/Symfony/Routing/ContentTreeLocationRemoteIdAware.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * File containing the ContentTreeLocationRemoteIdAware trait.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\Routing;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+
+trait ContentTreeLocationRemoteIdAware
```

Comment:
Imho it should be 'ContentTreeRootLocationIdAware'

Reply:
Also, this trait forces the class that implements it to depend on both Repo and ConfigResolver, which seems to be a lot of dependency... I wonder if it would be better off as part of a service instead ?